glyphs2fontir/src/source.rs

```diff
@@ -935,6 +934,23 @@ impl Work<Context, WorkId, WorkError> for GlyphIrWork {
     }
 }
 
+// this is intended as a drop-in replacement for a previous function of this
+// name. This will eventually need to be replaced with something that can handle
+// custom GlyphData.xml files, as well as handle overrides that are part of the
+// glyph source.
+fn is_nonspacing_mark(codepoints: &HashSet<u32>, name: &GlyphName) -> bool {
+    fn glyph_data() -> &'static GlyphData {
+        static GLYPH_DATA: OnceLock<GlyphData> = OnceLock::new();
+        GLYPH_DATA.get_or_init(|| GlyphData::new(None).unwrap())
+    }
```

Comment:
Why do you need a wapper function? - just set data directly to GLYPH_DATA.get_or_init